### PR TITLE
Fix string formatting syntax error in Gemini module

### PR DESCRIPTION
**Pull request by lpsinger**

### CODE REVIEW

**Comment by keflavich** [comment]
Could you add a regression test?  This is a pretty easy-to-miss error in code review, but is not subtle in its effects!

File: astroquery/gemini/core.py [hunk 1]
```diff
@@ -341,7 +341,7 @@ def query_criteria(self, *rawqueryargs, coordinates=None, radius=0.3*units.deg,
                     raise ValueError("utc_date tuple should have two values")
                 if not isinstance(utc_date[0], date) or not isinstance(utc_date[1], date):
                     raise ValueError("utc_date tuple should have date values in it")
-                args.append("%s-%s" % utc_date[0].strftime("YYYYMMdd"), utc_date[1].strftime("YYYYMMdd"))
+                args.append("%s-%s" % (utc_date[0].strftime("YYYYMMdd"), utc_date[1].strftime("YYYYMMdd")))
```
Comment (bsipocz) [inline astroquery/gemini/core.py:344]: nit picking, but could change it to `.format()` from the old style formatting? had we insisted on it during the original review, this issue would have been smoked out.

**Review by bsipocz — APPROVED** [review]

File: astroquery/gemini/core.py (hunk 1 above)
Comment (lpsinger) [reply astroquery/gemini/core.py]: Done.

**Comment by lpsinger** [comment]
> Could you add a regression test?

Done.

**Comment by bsipocz** [comment]
Thanks @lpsinger!

**Pull request merged**